src/server/getters/get-categories.ts

```diff
@@ -42,32 +53,84 @@ async function getCategoriesPopularity(db: Knex, universe: string): Promise<Arra
   return groupedCategoryInfo.map((categoryInfo: CategoryPopularity): CategoryPopularity => ({ popularity: categoryInfo.popularity.toString(), category: categoryInfo.category }));
```

Comment:
I thought category popularity is going away. Why is it still here?
Shouldn't it be called CategoryByOI?

Reply:
It is going away! This is WIP. Just pushed an update. Still have to (at least) do the migrations, update get-categories API to use the new columns, and update unit tests